src/controller/AppExecutor.java

```diff
@@ -0,0 +1,26 @@
+package controller;
+
+import service.LFUCache;
+import service.News;
+
+public class AppExecutor {
+    private News news = new News();
+    private LFUCache cache = new LFUCache(5);
+
+    public void runApp() {
+        news.putToNewsList();
+
+        for (long i = 1L; i <= 10L; i++) {
```

Comment:
what is 10?? as previous
and also, why u chose long type?

Reply:
done